asf_heat_pump_suitability/getters/get_datasets.py

```diff
@@ -131,15 +133,23 @@ def get_df_spa_offgasgrid() -> pl.DataFrame:
     return df
 
 
-def get_df_listedbuildings(data_source_key: str) -> pl.DataFrame:
+def load_gdf_listed_buildings(nation: str, **kwargs) -> gpd.GeoDataFrame:
```

Comment:
I made the function slightly less generic. My thinking: With the data source key, you can technically put any key for any data source in the config in here. However, because the function is loading a geospatial file, this won't work for most of the datasets. Even when it does work, it would then not be loading listed buildings data as the function describes. The change here to add the `nation` keyword makes the function more specific.